test/AspNetCoreModule.Test/Framework/InitializeTestMachine.cs

```diff
@@ -289,7 +412,7 @@ private void RollbackIISApplicationhostConfigFile()
             }
         }
 
-        private void PreparePrivateANCMFiles()
+        private void PreparePrivateANCMFiles(bool isIISReady)
```

Comment:
This bool is only used once, just read IISConfigUtility.IsIISReady.

Reply:
fixed